src/safeposix/cage.rs

```diff
@@ -8,6 +8,7 @@ pub use super::syscalls::fs_constants::*;
 pub use super::syscalls::sys_constants::*;
 pub use super::syscalls::net_constants::*;
 use super::filesystem::normpath;
+use std::sync::atomic::{AtomicI32};
```

Comment:
this needs to be added to the interface for import, look at how we do this with other types elsewhere.

Reply:
Done.